screens/Scan/ScanScreen.tsx

```diff
@@ -15,6 +15,7 @@ import {isIOS} from '../../shared/constants';
 import {BannerNotificationContainer} from '../../components/BannerNotificationContainer';
 import {SharingStatusModal} from './SharingStatusModal';
 import {SvgImage} from '../../components/ui/svg';
+import {assign} from "xstate";
```

Comment:
This import is not getting used it seems. If added intentionally we should ideally avoid using xstate in UI components.

Reply:
It was added for some other component I guess. Its not used anyways. Removed it. Plz check